extensions/amp-list/0.1/amp-list.css

```diff
@@ -14,6 +14,13 @@
  * limitations under the License.
  */
 
+/**
+ * Show the error messaging container on fetch error.
+ */
```

Comment:
As things stand, we may need to force hide it when `load-more` is present because `load-more` currently has it's own error-handling UI. Load more's error-handling UI must be always at the bottom of the list.

Reply:
How does load-more currently interact with `[fallback]`? At a glance, it looks like fallback is still shown if a load-more fetch fails.

Reply:
Yeah this is fine because neither the show fallback or the show fetch error codepath will be called in load-more.